python/pyspark/instrumentation_utils.py

```diff
@@ -31,7 +31,6 @@
 
 
 def _wrap_function(class_name: str, function_name: str, func: Callable, logger: Any) -> Callable:
-
```

Comment:
Can we just disable this check rather than touch so many files? doesn't seem worth it

Reply:
Yeah, I have actually noticed that there are a lot of files that have touched, but I am a bit worried that if we don't upgrade in a timely manner, we will encounter more files that require formatting style when upgrading in the future. Unless we never upgrade again.

Reply:
OK you're saying we should 'bite the bullet' for 4.0 and take all the changes? OK